.github/workflows/nuget-audit.yml

```
@@ -4,16 +4,6 @@ on:
 env:
   DOTNET_NOLOGO: true
 jobs:
-  build:
-    name: Linux
-    runs-on: ubuntu-22.04    
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4.2.2
-      - name: Setup .NET SDK
-        uses: actions/setup-dotnet@v4.1.0
-        with:
-          global-json-file: global.json
-      - name: Restore
-        id: restore
-        run: dotnet restore src
+  call-nuget-audit:
+    uses: particular/shared-workflows/.github/workflows/nuget-audit.yml@main
```

Needs to be updated to match the new name:
```suggestion
    uses: particular/shared-workflows/.github/workflows/audit-nuget.yml@main
```